src/ws/mock-config/cockpit/cockpit-deprecated.conf

```diff
@@ -0,0 +1,10 @@
+[WebService]
+ProtocolHeader = X-Forwarded-Proto
+
+[Ssh-Login]
+command = mock-auth-command
+host = default-host
+allowUnknown = true
```

Comment:
Could you add a comment here that indicates this is the deprecated option being tested?

Reply:
Done!